ReactSkia/modules/platform/libcurl/RSkNetworkingModule.cpp

```diff
@@ -72,6 +86,44 @@ uint64_t RSkNetworkingModule::nextUniqueId() {
   return id;
 }
 
+void RSkNetworkingModule::processNetworkRequest(CURLM *curlMultiHandle) {
+  CURLMcode res = CURLM_OK;
+  CURLMsg *msg;
+  int stillAlive = 1;
+  int msgsLeft = -1;
+  do {
+    res = curl_multi_perform(curlMultiHandle, &stillAlive);
+    while((msg = curl_multi_info_read(curlMultiHandle, &msgsLeft))) {
+      if(msg->msg == CURLMSG_DONE) {
+        int requestId = 0;
+        CURL *curlHandle = msg->easy_handle;
+        curl_easy_getinfo(msg->easy_handle, CURLINFO_PRIVATE, &requestId);
+        struct NetworkRequest *networkRequest = connectionList_[requestId];
+        // completionBlock
+        std::scoped_lock lock(connectionListLock_);
+        if(networkRequest !=NULL && (networkRequest->curl_ != NULL)) {
+          if(!(networkRequest->useIncrementalUpdates_ && (networkRequest->responseType_ == "text") ))
+            sendData(networkRequest); 
+          sendEventWithName("didCompleteNetworkResponse", folly::dynamic::array(networkRequest->requestId_ , curl_easy_strerror(msg->data.result) , (msg->data.result) == CURLE_OPERATION_TIMEDOUT ? "true" : "false" ));
```

Comment:
IN case of success what curl_easy_strerro will return ? As per IOS code, if error code is null, they pass NULL in second argument.
Also please make sure result is CURL Code and not CURLM code.